src/index.html

```diff
@@ -1,13 +1,55 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+ <html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta
+        name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
-</html>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?
+family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,500;1,700&disp
+lay=swap"
+      rel="stylesheet"
+    >
+  </head>
+  <body >
+    <div class="card-product"
+        data-qa="card">
+      <img
+        class="card-product__imac_img"
+        src="images/imac.jpeg"
+        alt="imac"
+      >
+
+      <h2 class="card-product__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+      <p class="card-product__code">Код товара: 195434</p>
+
+      <div class="card-product__rating">
+        <div class="card-product__stars stars">
```

Comment:
Let's add modifier `stars--4` for styling purposes.
```suggestion
        <div class="card-product__stars stars stars--4">
```